views/ddox.docpage.dt

```diff
@@ -14,60 +16,79 @@ block ddox.title
 		- title = prefix ~ names.join("/") ~ " - multiple declarations";
 	
 block ddox.members
+	- bool multi_page = info.docGroups.length > 1;
 
-	- if (info.docGroups.length > 1)
-		ul
-			- foreach (dg; info.docGroups)
-				- auto itm = dg.members[0];
-				li #{itm.kindCaption} #{itm.nestedName}
+	- if (multi_page)
+		nav.page-nav
+			ul
+				- foreach (i, dg; info.docGroups)
+					- auto itm = dg.members[0];
+					li: a(href="\##{i}") #{dg.kinds.joiner("/")} #{itm.nestedName}
```

Comment:
That would possibly result in duplicate anchor names. Using the mangled name as a basis could work, but the question is if that would be much better, since any small change in a function or template signature would change the outcome (thus rendering permanent links nearly as useless as for simple indexes).